app/views/layouts/application.html.erb

```diff
@@ -19,6 +19,7 @@
     </div>
 
 <%= render "layouts/footer" %>
-
+<div id="anon-stats">
```

Comment:
It should be like this:

https://github.com/EFForg/sec/blob/d7120ce2b520e5ddc6873a20b52e78532a566457/app/views/layouts/application.html.erb#L53-L60